common/protocol/Ola.proto

```diff
@@ -403,6 +406,7 @@ service OlaServerService {
   rpc GetDeviceInfo (DeviceInfoRequest) returns (DeviceInfoReply);
   rpc GetCandidatePorts (OptionalUniverseRequest) returns (DeviceInfoReply);
   rpc ConfigureDevice (DeviceConfigRequest) returns (DeviceConfigReply);
+  rpc SetPluginState (PluginStateChange) returns (Ack);
```

Comment:
How about 

PluginStateChangeRequest

I know it's long, but it's much easier to keep track of the types if they have a Request / Reply suffix.